src/corefx/System.Globalization.Native/collation.cpp

```diff
@@ -407,6 +401,24 @@ const UCollator* GetCollatorFromSortHandle(SortHandle* pSortHandle, int32_t opti
     return pCollator;
 }
 
+extern "C" int32_t GlobalizationNative_GetSortVersion(SortHandle* pSortHandle)
```

Comment:
I don't think the perf is an issue here as the usage of this functionality is very low and if used it will be called just once to check if there is re-indexing needed for the used data.